apps/studio/components/interfaces/Integrations/Vault/Secrets/EditSecretModal.tsx

```
@@ -40,22 +36,33 @@ const SecretSchema = z.object({
 
 const formId = 'edit-vault-secret-form'
 
-const EditSecretModal = ({ visible, secret, onClose }: EditSecretModalProps) => {
-  const [showSecretValue, setShowSecretValue] = useState(false)
+export const EditSecretModal = () => {
   const { data: project } = useSelectedProjectQuery()
+
+  const { data: secrets = [], isSuccess } = useVaultSecretsQuery({
+    projectRef: project?.ref!,
```

Biome (2.3.14)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)